ansible-test/my-playbook.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: local-test
+  remote_user: root
```

Comment:
The keyword `sudo` has long been deprecated.

Use `become: yes` then it is safe to omit the `sudo` params later.